modules/react/_examples/stories/examples/SidePanelWithNavigation.tsx

```diff
@@ -0,0 +1,238 @@
+    backgroundColor: colors.soap500,
+  },
+});
+
+const StyledLink = styled(Subtext.as('a'))<StyledType>({
+  color: colors.blackPepper300,
+  ':hover': {
+    backgroundColor: colors.soap500,
+  },
+});
+
+const forDesigners = [
+  {value: 'Understanding Canvas', linkValue: '#UnderstandingCanvas'},
+  {value: 'Customizing Canvas', linkValue: '#CustomizingCanvas'},
+  {value: 'Contributing to Canvas', linkValue: '#ContributingtoCanvas'},
+  {value: 'Resources', linkValue: '#Resources'},
+];
+
+const forDevelopers = [
+  {value: 'Using Canvas', linkValue: '#UsingCanvas'},
+  {value: 'Contributing', linkValue: '#Contributing'},
+  {value: 'Resources', linkValue: '#Resources'},
+];
+
+const gettingStarted = [
+  'Extended Systems',
+  'Canvas Glossary',
+  'Frequently Asked Questions',
+  'Canvas Support',
+];
+
+export const WithNavigation = () => {
+  const {labelProps} = useSidePanel();
+
+  return (
+    <Flex height={800} position="relative" backgroundColor="frenchVanilla100">
+      <StyledPanel touched={true} variant="alternate">
+        <Flex flexDirection="column" rowGap="zero" paddingTop="m">
+          <Flex.Item>
+            <Expandable padding="zero">
+              <StyledExpandable paddingY="s" paddingX="m" textAlign="center">
+                <SystemIcon icon={rocketIcon} marginX="xxs" />
+                <Expandable.Title padding="zero">
+                  <BodyText
+                    as="h4"
+                    size="small"
+                    fontWeight="bold"
+                    color="blackpepper300"
+                    marginY="zero"
+                  >
+                    Getting Started
+                  </BodyText>
+                  <Subtext size="small" color="blackpepper300" marginY="zero">
+                    Guidance for beginners to pros
+                  </Subtext>
+                </Expandable.Title>
+                <Expandable.Icon iconPosition="end" />
+              </StyledExpandable>
+              <Expandable.Content paddingY="zero" paddingX="zero">
+                <StyledLink
+                  as="a"
+                  href="#"
+                  size="large"
+                  color={colors.blackpepper300}
+                  textDecoration="none"
+                  marginY="zero"
+                  display="flex"
+                  paddingTop="10px"
+                  paddingBottom="9px"
+                  paddingLeft="xxl"
+                >
+                  Introduction
+                </StyledLink>
+                <Expandable paddingBottom="9px" paddingX="zero" paddingTop="zero">
+                  <StyledExpandable
+                    alignItems="center"
+                    paddingLeft="xxl"
+                    paddingTop="10px"
+                    paddingBottom="zero"
+                  >
+                    <Expandable.Icon iconPosition="start" size="16px" padding="zero" />
+                    <Expandable.Title>
+                      <Subtext
+                        as="a"
```

Comment:
Anchor element cannot exist inside of a parent <button> element. 
![links cannot be inside buttons](https://user-images.githubusercontent.com/40372497/228909243-23df6af5-975e-4d35-aaed-3222597ccbab.png)